react/week1/todo-classes/src/components/addItems.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+export default class AddItem extends Component {
+    
+        state =  {
+            currentItem:[
+            {
+                title: "Get out of bed, Wed Sep 13 2017",
+            },
+            {
+                title: "Brush teeth, Sep 14 2017",
+            },
+            {
+                title: "Eat breakfast, Fri Sep 15 2017"
+            }
+        ]};        
+
+   render() { 
+        return (
+            <div>             
+            {this.state.currentItem.map((todo, index) => (
```

Comment:
please move your array map into ul tag because now you are creating a ul tag with a li tag for each todo items, but the correct way of doing it is to create a li tag for each todo item.

Reply:
I moved map into ul tag

Reply:
Thanks for your feedback